docs/changehistory/NextVersion.md

```diff
@@ -298,6 +298,10 @@ In addition to upgrading iTwin.js core dependencies to `4.0`, there are some oth
 - Upgrade [iTwinUI](https://github.com/iTwin/iTwinUI) from v1 to v2.
 - `@itwin/presentation-backend`, `@itwin/presentation-common` and `@itwin/presentation-frontend` have new peer dependency `@itwin/ecschema-metadata`.
 
+### Removed deprecated items
```

Comment:
We already have a "Deprecated API removals" section under "Breaking changes".
Spelling: "polymorphicly" => "polymorphically".